### PR TITLE
[v9.2.0] Update podspec and CHANGELOG for 9.2.0

### PR DESCRIPTION
**Pull request by ncooke3**
### Context
- Following the merge of `metrics-main` (#77), this PR preps the podspec and CHANGELOG for the 9.2.0 release

### CODE REVIEW

**Comment by ncooke3** [comment]
I'll wait for LGTM and green CI before merging. Then, I'll stage the podspec on SpecsStaging.

**Review by chliangGoogle — APPROVED** [review]

**Review by paulb777 — APPROVED** [review]
Good to merge after clean nightlies and global Tap

**Pull request merged**
